tests/layout-checks.py

```diff
@@ -175,7 +184,7 @@ def check_empty_check():
         # z/tempest is a fake project with no check queue
         if project['name'] == 'z/tempest':
             continue
-        check_jobs = collect_pipeline_jobs(project, 'check')
+        check_jobs = collect_pipeline_jobs(project, 'check_github')
```

Comment:
I think we will want to ignore experimental and periodic pipelines, if we ever add them to our layout. It makes sense to me to be explicit about which pipelines we want to enforce rules on.

But rather than changing this, I'm in favor of renaming our pipelines to remove the trailing "_github" since we no longer have a gerrit connection.